CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ function(packageProject)
   cmake_parse_arguments(
     PROJECT
     ""
-    "NAME;VERSION;INCLUDE_DIR;INCLUDE_DESTINATION;BINARY_DIR;COMPATIBILITY;VERSION_HEADER"
+    "NAME;VERSION;INCLUDE_DIR;INCLUDE_DESTINATION;BINARY_DIR;COMPATIBILITY;VERSION_HEADER;LIBRARY_NAMESPACE"
```

Comment:
I wonder if we could use the shorter named argument `NAMESPACE`, as it should already be unambiguous.

Reply:
I will change that